### PR TITLE
CP-919: First Pass at Connected Account Parent Template

### PR DESCRIPTION
**Pull request by Tankanow**

### CODE REVIEW

**Comment by jeremyhahn** [comment]
:+1:

File: services/connected_account.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    Description: |
+      CloudZero AWS AccountID; for cross-account Role Access
+  ReactorSQSQueueUrl:
+    Type: String
+    Default: Does not Exist Yet
+    Description: |
+      CloudZero Reactor Queue Url; receives callbacks from this template
+
+Resources:
+  Disovery:
```
Comment (killthrush) [inline services/connected_account.yaml]: sp. "discovery"

File: services/connected_account.yaml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        - Key: cloudzero
+          Value: cloudzero
+      TemplateURL: https://s3.amazonaws.com/cz-provision-account/services/latest/discovery.yaml
+
+  ResourceOwnerAccount:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Tags:
+        - Key: cloudzero
+          Value: cloudzero
```
Comment (killthrush) [inline services/connected_account.yaml]: this k:v pair looks dubious...  probably should say cloudzero *\something*, but what exactly are we trying to communicate?

Comment (Tankanow) [reply services/connected_account.yaml]: I agree ... I did a quick googling for AWS 3rd Party Tagging best practices but didn't find much. I want to make sure our customers can easily find our resources.

Comment (Tankanow) [reply services/connected_account.yaml]: Maybe Erik has an opinion.

File: services/connected_account.yaml (hunk 1 above)
Comment (Tankanow) [reply services/connected_account.yaml]: Fixed!

**Pull request merged**